arangopipe/arangopipe/arangopipe_storage/arangopipe_admin_api.py

```diff
@@ -388,15 +392,15 @@ def create_enterprise_ml_graph(self, db_replication_factor):
         for edge, fromv, tov in zip(edge_names, from_list, to_list):
             if not self.emlg.has_edge_definition(edge):
```

Comment:
we could possibly check if the collection exists independent of if the graph has the edge definition? For example something along the lines of:
```
        if not db.has_collection(edge):
            db.create_collection(
                edge, edge=True, replication_factor=db_replication_factor
            )
        if not emlg.has_edge_definition(edge):
            emlg.create_edge_definition(
                edge_collection=edge,
                from_vertex_collections=[fromv],
                to_vertex_collections=[tov],
            )
```

What do you think?

Reply:
Very good point